src/restrictededitingmodeediting.js

```diff
@@ -92,6 +95,28 @@ export default class RestrictedEditingModeEditing extends Plugin {
 		} );
 	}
 
+	/**
+	 * Enables command with given `commandName` in restricted editing mode.
+	 *
+	 * @param {String} commandName Name of the command to enable.
+	 */
+	enableCommand( commandName ) {
+		const command = this.editor.commands.get( commandName );
+
+		if ( !command ) {
```

Comment:
Seems unnecessary. If it's going to fail, it's going to fail in the very next line so it's super easy to debug.

Reply:
Done.